x-pack/legacy/plugins/spaces/server/lib/get_spaces_usage_collector.ts

```diff
@@ -131,19 +130,17 @@ export function getSpacesUsageCollector(deps: CollectorDeps) {
     fetch: async (callCluster: CallCluster) => {
       const xpackInfo = deps.xpackMain.info;
       const available = xpackInfo && xpackInfo.isAvailable(); // some form of spaces is available for all valid licenses
-      const enabled = deps.config.get('xpack.spaces.enabled');
-      const spacesAvailableAndEnabled = Boolean(available && enabled);
 
       const usageStats = await getSpacesUsage(
         callCluster,
-        deps.config.get('kibana.index'),
+        deps.kibanaIndex,
         deps.xpackMain,
-        spacesAvailableAndEnabled
+        available
       );
 
       return {
         available,
-        enabled: spacesAvailableAndEnabled, // similar behavior as _xpack API in ES
+        enabled: available,
```

Comment:
It's not technically necessary to have both `available` and `enabled` here, but I chose to keep it in so that telemetry consumers can have a consistent schema when searching for spaces-related statistics.